### PR TITLE
twis40

### PR DESCRIPTION
**Pull request by larsbergstrom**
r? @Manishearth @jdm

[Review on Reviewable](https://reviewable.io/reviews/servo/blog.servo.org/38)

### CODE REVIEW

File: _posts/2015-11-02-twis-40.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+categories:
+---
+
+In the [last week](https://github.com/pulls?page=1&q=is%3Apr+is%3Amerged+closed%3A2015-10-26..2015-11-02+user%3Aservo),
+we landed 97 PRs in the Servo organization's repositories.
+
+###Notable additions
+
+ - Emily Dunham and jdm did [some](https://github.com/servo/servo.org/pull/3) [work](https://github.com/servo/servo-starters/pull/9) getting https://servo.org/starters working!
+ - Manish added support for [`@bors-servo: try`](https://github.com/servo/saltfs/pull/141) for people who don't have full reviwer priviledges in Servo
```
Comment (Manishearth) [inline _posts/2015-11-02-twis-40.md]: spelling: reviewer

File: _posts/2015-11-02-twis-40.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+summary:    What's up with Servo for the week of 27 Oct 2015 - 02 Nov 2015
+categories:
+---
+
+In the [last week](https://github.com/pulls?page=1&q=is%3Apr+is%3Amerged+closed%3A2015-10-26..2015-11-02+user%3Aservo),
+we landed 97 PRs in the Servo organization's repositories.
+
+###Notable additions
+
+ - Emily Dunham and jdm did [some](https://github.com/servo/servo.org/pull/3) [work](https://github.com/servo/servo-starters/pull/9) getting https://servo.org/starters working!
```
Comment (eefriedman) [inline _posts/2015-11-02-twis-40.md]: `https://servo.org/starters` doesn't work.

Comment (Manishearth) [inline _posts/2015-11-02-twis-40.md]: No https

**Comment by jdm** [comment]
Looks good to me!

**Pull request merged**